src/script.js

```diff
@@ -441,6 +461,7 @@ module.exports = {
   isMultisigOutput: isMultisigOutput,
   isNullDataOutput: isNullDataOutput,
 
+  parseMultisigScript: parseMultisigScript,
```

Comment:
@afk11 true.... `m-of-n` makes sense... now that I know it,  but it does require incantation to remember what it means.